dir/path.go

```diff
@@ -52,14 +69,19 @@ func checkError(err error) {
 	}
 }
 
-// Config returns the path of config.json
+// Config returns the ready-only path of config.json
 func (p *PathManager) Config() string {
 	path, err := p.ConfigFS.GetPath(ConfigFile)
 	checkError(err)
 	return path
 }
 
-// LocalKey returns path of the local private key and it's certificate
+// ConfigForWrite returns the writable path of config.json
+func (p *PathManager) ConfigForWrite() string {
```

Comment:
WriteLevel is better to be an argument to this func instead of a var in the package. See below.

Reply:
Done.